tests/layer_tests/common/utils/common_utils.py

```diff
@@ -26,6 +26,10 @@ def generate_ir(coverage=False, **kwargs):
             params.extend(("-b", str(value)))
         elif key == "k":
             params.extend(("-k", str(value)))
+        # for FP32 set explicitly compress_to_fp16=False,
```

Comment:
It looks that we can run with `FP16` or `FP32` mode using two ways by setting `compress_to_fp16` or `precision`? Do we need both ways? Can they contradict to each other?

Reply:
They cannot contradict, layer tests have only one `precision` argument, `compress_to_fp16` is internal and it's set based on precision: 
https://github.com/openvinotoolkit/openvino/blob/67afe170d2652f37e7f2a0af3f7fb04825073d11/tests/layer_tests/common/layer_test_class.py#L43-L47

user cannot run layer tests with defined compress_to_fp16 only, it should use precision

Reply:
Got it.